src/Accessor.elm

```diff
@@ -393,25 +345,35 @@ and returns the data `structure` with the focusView field changed by applying
 the function to the existing value.
 
     { foo = { qux = 0 } }
-        |> mapOver (Record.foo << Record.qux) ((+) 1)
+        |> over (Record.foo << Record.qux) ((+) 1)
     --> { foo = { qux = 1 } }
 
 -}
-mapOver :
-    TraversalConsume structure focus focusView
+over :
+    Traversal_ structure focus focusView
```

Comment:
The `_` postfix comes from the Haskell equivalent `Lens s t a b` & `Lens' s a` which is just referenced as the "simplified" Lens...

Not sure what `Consume` means so I dropped it for all the "simplified" optics.